internal/locale/locales/en-us.yaml

```diff
@@ -219,6 +219,8 @@ logout_description:
   other: Logout
 logged_out:
   other: You have been logged out
+logout_still_have_api_token:
+  other: You still have an API token configured (environment variable or Google Cloud). You will not be completely logged out until you remove it.
```

Comment:
```suggestion
  other: Logout is incomplete because you have authenticated with an environment variable. To complete logout please unset the '[ACTIONABLE]ACTIVESTATE_API_KEY[/RESET]' environment variable.
```